staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/micro_time_test.go

```diff
@@ -57,6 +57,9 @@ func TestMicroTimeUnmarshalYAML(t *testing.T) {
 		result MicroTime
 	}{
 		{"t: null\n", MicroTime{}},
+		{"t: 1998-05-05T05:05:05Z\n", MicroTime{Date(1998, time.May, 5, 5, 5, 5, 0, time.UTC).Local()}},
+		{"t: 1998-05-05T05:05:05.123Z\n", MicroTime{Date(1998, time.May, 5, 5, 5, 5, 123000000, time.UTC).Local()}},
+		{"t: 1998-05-05T05:05:05.123456Z\n", MicroTime{Date(1998, time.May, 5, 5, 5, 5, 123456000, time.UTC).Local()}},
```

Comment:
please also add the test cases for Marshal / output to test the new change.

Reply:
@yliaog I expanded the marshaling tests